website/en/docs/types/functions.md

```diff
@@ -348,5 +351,16 @@ method(function(a: number, b: number) {
 });
 ```
 
+Neither will this:
+
+```js
+function method(obj: Function) {
+  obj = 10;
+}
+
+method(function(a: number, b: number) {
+  // ...
+});
+
```

Comment:
Close the triple backticks here

Reply:
good catch! sorry I wasn't actually able to run the site generation locally.